lib/Elevate/Blockers/EA4.pm

```diff
@@ -87,6 +87,7 @@ sub _get_incompatible_packages ($self) {
     }
 
     if (@imunify_pkgs) {
+        Elevate::StageFile::remove_from_stage_file('ea4_imunify_packages');
```

Comment:
It does fix a bug from RE-313 (the main story for this case).  I learned that `update_stage_file` will duplicate content instead of replacing it, so it is necessary to execute `remove_form_stage_file` here and in the other spot that I added it in this PR in order to prevent `--check` from adding duplicate content each time it is executed